Added wiz_InitInfo

taylor-an opened the pull request.

Commit:
Added wiz_InitInfo

TeddyWiz committed:
modiry mcu_init.c  cube mx version error  #ifdef USE_STDPERIPH_DRIVER

TeddyWiz committed:
modiry cubeMX board init and change delay function

TeddyWiz committed:
remove extern spi move define spi board.h

TeddyWiz committed:
remove extern  add init sturcture

Commit:
Merge branch 'wizchip_init_fuc-taylor' into wizchip_init_fuc-teddy

Commit:
Merge pull request #9 from WIZnet-ioLibrary/wizchip_init_fuc-teddy

Wizchip init fuc teddy

taylor-an commented on TrueStudio/W6100EVB-Loopback/Src/board_init.c:
```diff
@@ -33,37 +33,38 @@ void BoardInitialze(void)
-	myW6100.spi_wbuf = NULL;
+	W6100_Info.spi_rb = spiReadByte;
+	W6100_Info.spi_wb = spiWriteByte;
+	W6100_Info.spi_rbuf = NULL;
```
NULL 보다 비었다는 Message를 출력하는 Dummy 함수로..

taylor-an replied:
https://github.com/WIZnet-ioLibrary/W6100EVB-Loopback/pull/8/commits/8da5ed70c95bf58c2411e36711902001988170a2

taylor-an commented on TrueStudio/W6100EVB-Loopback/Src/board_init.c:
```diff
@@ -33,37 +33,38 @@ void BoardInitialze(void)
+	W6100_Info.bus_rd = busReadByte;
+	W6100_Info.bus_wd = busWriteByte;
+	W6100_Info.bus_rbuf = NULL;
+	W6100_Info.bus_wbuf = NULL;
```
NULL 보다 비었다는 Message를 출력하는 Dummy 함수로..

taylor-an replied:
https://github.com/WIZnet-ioLibrary/W6100EVB-Loopback/pull/8/commits/8da5ed70c95bf58c2411e36711902001988170a2

taylor-an commented on TrueStudio/W6100EVB-Loopback/Src/board_init.c:
```diff
@@ -33,37 +33,38 @@ void BoardInitialze(void)
-	myW6100.resetAssert = resetAssert;
-	myW6100.resetDeassert = resetDeassert;
-	
+	//wizchip_init_Reset_Func(resetAssert, resetDeassert);
```
이런 이름의 함수가 있나요??

TeddyWiz replied:
제가 만들었다가 지웠는데 남아있었나보네요

taylor-an replied:
https://github.com/WIZnet-ioLibrary/W6100EVB-Loopback/pull/8/commits/8da5ed70c95bf58c2411e36711902001988170a2

taylor-an commented on TrueStudio/W6100EVB-Loopback/Inc/board_init.h:
```diff
@@ -1,5 +1,7 @@
+//#define USE_STDPERIPH_DRIVER
```
Merge하기 전에 제거 될 Line..
VS Code에서 Pre-defined를 처리 못하는것 같아서 추가

taylor-an commented on TrueStudio/W6100EVB-Loopback/Inc/mcu_init.h:
```diff
@@ -1,3 +1,5 @@
+//#define USE_STDPERIPH_DRIVER
```
Merge하기 전에 제거 될 Line..
VS Code에서 Pre-defined를 처리 못하는것 같아서 추가

taylor-an commented on TrueStudio/W6100EVB-Loopback/Src/board_init.c:
```diff
@@ -12,7 +12,7 @@ void spi_set_func(SPI_HandleTypeDef *SPI_n)
 #endif
 
-extern wiz_InitInfo myW6100;
+wiz_InitInfo W6100_Info;
```
한번만 쓰니깐 Local 변수로 쓸까요?

TeddyWiz replied:
로컬 변수로 쓰는게 나을것 같아요

taylor-an replied:
https://github.com/WIZnet-ioLibrary/W6100EVB-Loopback/pull/8/commits/8da5ed70c95bf58c2411e36711902001988170a2

Commit:
Added spiDummyReadBurst()
Added spiDummyWriteBurst()
Modified print_network_information()
Added print_ipv6_addr()
